__tests__/endpoint.test.js

```diff
@@ -332,3 +332,72 @@ describe('POST/api/articles/:article_id/comment', () => {
   });
 
 })
+
+describe('PATCH/api/articles/:article_id', () => {
+  test('PATCH:200 sends a single article object to the client with vote increment', () => {
+
+    return db.query('SELECT * FROM articles WHERE article_id = 1').then ((result)=>{
+
+      const oldVote = result.rows[0].votes
+
+      const testVote = { vote_increment : 6 }
+
+    return request(app)
+      .patch("/api/articles/1").send(testVote)
+      .expect(200)
+      .then((response) => {
+
+        const newVote = oldVote + testVote.vote_increment
+
+        const article= response.body.article
+         
+        expect(article.length).toBe(2)
```

Comment:
Why is the length 2 here? We should get a single updated article back. This should be an object (not an array).